app/assets/javascripts/controllers/container_live_dashboard/container_live_dashboard_controller.js

```diff
@@ -0,0 +1,287 @@
+/* global miqHttpInject */
+
+miqHttpInject(angular.module('containerLiveDashboard', ['ui.bootstrap', 'patternfly', 'patternfly.charts']))
+  .controller('containerLiveDashboardController', ['$scope', 'pfViewUtils', '$location', '$http', '$interval', '$timeout', '$window',
+  function ($scope, pfViewUtils, $location, $http, $interval, $timeout, $window) {
+    var tenant = '_ops';
+
+    $scope.filtersText = '';
+    $scope.definitions = [];
+    $scope.items = [];
+    $scope.tags = {};
+    $scope.tagsLoaded = false;
+
+    $scope.applied = false;
+    $scope.filterChanged = true;
+    $scope.viewGraph = false;
+    $scope.chartData = {};
+
+    $scope.timeRanges = [
+      {title: _("Hours"), value: 1},
+      {title: _("Days"), value: 24},
+      {title: _("Weeks"), value: 168},
+      {title: _("Months"), value: 672}
+    ];
+
+    $scope.timeFilter = {
+      time_range: 24,
+      range_count: 2,
+      date: new Date()
+    };
+
+    $scope.dateOptions = {
+      autoclose: true,
+      todayHighlight: true,
+      orientation: 'bottom'
+    };
+
+    $scope.countDecrement = function() {
+        if($scope.timeFilter.range_count > 1) {
```

Comment:
Please keep the spacing consistent... (suddenly 4 instead of 2 - in multiple functions).

Also, space after if ;).